webapp/channels/src/components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -239,66 +375,77 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
-                className={'more-modal__row clickable ' + rowSelected}
-                onClick={() => onAdd(option)}
-                onMouseMove={() => onMouseMove(option)}
-            >
-                <ProfilePicture
-                    src={Client4.getProfilePictureUrl(option.id, option.last_picture_update)}
-                    status={this.props.userStatuses[option.id]}
-                    size='md'
-                    username={option.username}
-                />
-                <div className='more-modal__details'>
-                    <div className='more-modal__name'>
-                        <span>
-                            {displayName}
-                            {option.is_bot && <BotTag/>}
-                            {isGuest(option.roles) && <GuestTag className='popoverlist'/>}
-                            {displayName === option.username ?
-                                null :
+            const userMapping: Record<string, string> = {};
+            for (let i = 0; i < ProfilesInGroup.length; i++) {
+                userMapping[ProfilesInGroup[i]] = 'Already in channel';
+            }
+            const displayName = displayUsername(option, this.props.teammateNameDisplaySetting);
+            return (
+                <div
+                    key={option.id}
+                    ref={isSelected ? this.selectedItemRef : option.id}
+                    className={'more-modal__row clickable ' + rowSelected}
+                    onClick={() => onAdd(option)}
+                    onMouseMove={() => onMouseMove(option)}
+                >
+                    <ProfilePicture
+                        src={Client4.getProfilePictureUrl(option.id, option.last_picture_update)}
+                        status={this.props.userStatuses[option.id]}
+                        size='md'
+                        username={option.username}
+                    />
+                    <div className='more-modal__details'>
+                        <div className='more-modal__name'>
+                            <span>
+                                {displayName}
+                                {option.is_bot && <BotTag/>}
+                                {isGuest(option.roles) && <GuestTag className='popoverlist'/>}
+                                {displayName === option.username ?
+                                    null :
+                                    <span
+                                        className='ml-2 light'
+                                        style={{fontSize: '12px'}}
+                                    >
+                                        {'@'}{option.username}
+                                    </span>
+                                }
                                 <span
-                                    className='ml-2 light'
-                                    style={{fontSize: '12px'}}
+                                    style={{position: 'absolute', right: 20}}
```

Comment:
Should we be using inline styles like this? Maybe you can use styled components here?